### PR TITLE
Missing imports and a few typos

### PR DESCRIPTION
**Pull request by sjrl**
## Description
I noticed some missing imports and a few typos so I made a few changes. Please let me know if this helpful!

## Status

- [x] Code base linted
- [ ] Ready to go

### CODE REVIEW

File: qcelemental/molutil/align.py [hunk 1]
```diff
@@ -263,6 +263,7 @@ def _plausible_atom_orderings_wrapper(
             print(atomfmt2.format(auniq[at][:6], *[ageom[at][i] - rgeom[at][i] for i in range(3)]))
 
     if do_plot:
+        # TODO Missing import
```
Comment (loriab) [inline qcelemental/molutil/align.py]: yeah, this is a relic from when I was developing the code in jupyter and had the target, before, and after mols pop up for sanity checking. possibly this should be hidden further now.

**Comment by loriab** [comment]
CI fails look to be pubchem timeouts. They happen.

**Comment by dgasmith** [comment]
`TYPE_CHECKING` is always hand written and should be validated by `mypy`. Always curious if we should enforce `mypy` checking which will add quite a bit of complexity.

**Comment by loriab** [comment]
gtk, I thought some `TYPE_CHECKING` blocks materialized when I ran `make format`. I don't feel strongly on `mypy`. If it was easy to enforce by tool, I'd be willing. But I'm also satisfied by semiannual `mypy` passes.

**Pull request merged**